test/cluster-store.js

```diff
@@ -19,7 +19,6 @@ if (cluster.isWorker) {
 
 describe('clustered socket.io', function() {
   before(setupWorkers);
-  after(stopWorkers);
```

Comment:
In short: I doubt it was ever needed, I suspect it was just an attempt to "cleanup", but the workers are always cleaned up on parent death.